packages/flutter_tools/lib/src/macos/build_macos.dart

```diff
@@ -30,9 +30,10 @@ Future<PrebuiltMacOSApp> buildMacOS({
     projectDir: flutterProject.directory,
     buildDir: flutterProject.dartTool.childDirectory('flutter_build'),
     defines: <String, String>{
-      kBuildMode: buildInfo.isDebug == true ? 'debug' : 'release',
+      // TODO(jonahwilliams): support other build types.
+      kBuildMode: 'debug',
```

Comment:
Won't this cause the Xcode project itself to be built in Debug mode? That would be very problematic for at least one early adopter.

I'd like to have `--release` still be available, but just use the debug framework for now.

Reply:
No, this only controls how the kernel is built. The xcode release mode is set below on L57, and in the generated xcconfig